### ISSUE TITLE
Documentation out of date or irrelevant

### ISSUE THREAD

**Issue by shadowae** [reactjs/react-docgen#202]
https://facebook.github.io/react/docs/components-and-props.html

Referring to this definition of documentation of how components and classes are defined, I find it hard to follow the doc when its in diff format. And I'm getting difficulty using the api to extract the information. 

Maybe the documentation isn't really easy for me to follow but some direction will help

**Comment by fkling** [comment]
It's not clear to me what exactly you want. Could you provide an example?
